### ISSUE TITLE
enable deletion of single url 

### ISSUE THREAD

**Issue by AyushSharma72** [TherkuTech/tturl#63]
right now there is not option by which user can delete a single url i want to add an cross symbol to each url in the history 
so that the user can delete a specific url 

![image](https://github.com/user-attachments/assets/ee7a2af1-9279-45dc-b49a-f603350420a3)

please assign this issue to me @muruga21

**Comment by muruga21** [comment]
@AyushSharma72 yeah dude.. work on it.. dont delete the id from database.. add a attribute called deleted with type bool in the schema.. and set the value to true = "deleted" ..  and while redirecting to the corresponding page.. make sure when you try to use deleted ur it should not redirect
